rest-model/src/main/java/org/jboss/pnc/rest/restmodel/BuildExecutionConfigurationRest.java

```diff
@@ -95,6 +116,20 @@ public BuildExecutionConfiguration toBuildExecutionConfiguration() {
         );
     }
 
+    public BuildConfigurationAudited toBuildConfigurationAudited() {
+        BuildConfigurationAudited buildConfigAudited = new BuildConfigurationAudited();
```

Comment:
@matejonnet  Ok I see your point, thanks. This conversion is done just to mimic the relation that the BuildTask has with the BuildConfiguration, in order to use the rsql query also with the in memory Builds (the running ones).What if I convert to BuildConfigurationAuditedRest? There is already a UserRest inside the BuildExecutionConfigurationRest object?